app/src/main/res/layout/activity_welcome.xml

```diff
@@ -10,11 +10,25 @@
         android:layout_height="match_parent"
         android:fadingEdge="none" />
 
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/overflow_button_dimen"
+        android:layout_marginTop="@dimen/standard_gap"
+        android:layout_marginRight="@dimen/standard_gap"
+        android:text="@string/welcome_skip_button"
+        android:id="@+id/welcomeYesButton"
```

Comment:
Maybe it would be worth renaming this to something more appropriate e.g. giving it an id of `welcomeSkipButton`